### PR TITLE
Remove accelbyte and add Epic Games to sponsors. [RESOLVES #1692][RESOLVES #1690]

### LINKED ISSUE #1690: [SITE][FEATURE] Remove inactive member logos
## Describe the feature you'd like available on the website

Remove AccelByte logo from https://www.o3de.org/

## Proposed implementation

Remove AccelByte logo from https://www.o3de.org/

## Available alternatives

N/A

### LINKED ISSUE #1692: [SITE][FEATURE] Add Epic Games as Sponsor Logo
## Describe the feature you'd like available on the website

Add the Epic Games logo as a sponsor

Logo: https://upload.wikimedia.org/wikipedia/commons/thumb/3/31/Epic_Games_logo.svg/1032px-Epic_Games_logo.svg.png?20180409155704

### PR DESCRIPTION
**Pull request by sptramer**
Changes sponsors for the main page, removing an inactive sponsor and adding Epic Games as a premier partner of the O3DF.

Signed-off-by: Stephen Tramer <169061+sptramer@users.noreply.github.com>

**MUST BE MERGED JULY 20, 7AM - 9AM AT THE SAME TIME AS #1799**

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: layouts/partials/home/partners.html
Comment (AMZN-daimini) [inline layouts/partials/home/partners.html:19]: Please add this in the right position (between AWS and Huawei). Partners of the same category are listed in alphabetical order.

File: assets/sass/custom.sass
Comment (AMZN-daimini) [inline assets/sass/custom.sass:985]: nit: these were originally in alphabetical order too, can you fix them? Thanks :D

**Review by AMZN-daimini — APPROVED** [review]

File: layouts/partials/home/partners.html
Comment (uoduckswtd1) [reply layouts/partials/home/partners.html:19]: +1. Agreed!

**Review by uoduckswtd1 — APPROVED** [review]
I approve, provided that logos are listed in alpha order within their respective groups, and that AccelByte is removed.

**Comment by sptramer** [comment]
Partner logos reordered, requesting final validation and merge from a maintainer.

**Comment by AMZN-daimini** [comment]
Logo is absent in the deploy preview
image

**Comment by sptramer** [comment]
However, it renders perfectly for me:
image

Firefox 91.11.0esr (64-bit), Windows 10

**Review by AMZN-daimini — APPROVED** [review]
Thanks for fixing it, looks good now!
image

**Review by chanmosq — APPROVED** [review]
Everything else seems good!

**Pull request merged**
